net-analyzer/zeek/zeek-3.0.1.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="The Zeek Network Security Monitor"
+HOMEPAGE="https://www.zeek.org"
+SRC_URI="https://www.zeek.org/downloads/${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="curl debug geoip2 ipsumdump ipv6 jemalloc kerberos +python sendmail \
+	static-libs tcmalloc +tools +zeekctl"
+
+RDEPEND=">=sys-libs/glibc-2.10
```

Comment:
All versions in gentoo are newer then 2.10, you can remove this implicit system dependency

Reply:
Done